syntax = "proto3";

// The package can be anything,
// but it is recommended to use the reverse domain name of the organization.
// For example, if your organization owns "example.com", you might use "com.example"
// using a reverse domain name ensures that your package name is globally unique,
// which helps avoid naming collisions when integrating with other systems.
package grpc.main;

service Main {
  rpc transcribe_audio (transcription_request) returns (transcription_reply) {}
}

message transcription_request {
    bytes audio_base64 = 1;
    string language = 2;
}

message transcription_reply {
    string transcription = 1;
}
